asteroid/metrics.py

```diff
@@ -220,15 +221,16 @@ class WERTracker:
             See librimix/ConvTasNet recipe.
     """
 
-    def __init__(self, model_name, trans_df):
+    def __init__(self, model_name, trans_df, use_gpu=1):
```

Comment:
use_gpu should be a boolean, so default to True if you want. Plus add arg to docstring.

Reply:
fixed